Content.Shared/_DV/CosmicCult/SharedMonumentSystem.cs

```diff
@@ -0,0 +1,166 @@
+using Content.Shared._DV.CosmicCult;
+using Content.Shared._DV.CosmicCult.Components;
+using Content.Shared._DV.CosmicCult.Prototypes;
+using Content.Shared.Actions;
+using Content.Shared.Movement.Components;
+using Content.Shared.Nutrition.Components;
+using Content.Shared.UserInterface;
```

Comment:
```suggestion
using Content.Shared._DV.CosmicCult.Components;
using Content.Shared._DV.CosmicCult.Prototypes;
using Content.Shared.Actions;
using Content.Shared.UserInterface;
```
Also as per the comment on the first line that was marked resolved and that VSC won't let me extend this comment over, `using Content.Shared._DV.CosmicCult;` also needs to go